File: 8-kyu/basic-mathematical-operations/basic-mathematical-operations.cs
namespace Solution
{
  public static class Program
  {
    public static double basicOp(char operation, double value1, double value2)
    {
      double result = 0;
      
      if(operation.ToString() == "+") result = value1 + value2;      
      if(operation.ToString() == "-") result = value1 - value2;      
      if(operation.ToString() == "*") result = value1 * value2;     
      if(operation.ToString() == "/") result = value1 / value2;
​
      return result;
    }
  }
}

File: 8-kyu/basic-mathematical-operations/basic-mathematical-operations.py
def basic_op(operator, value1, value2):
    
    match operator:
        case "+":
            result = value1 + value2
        case "-":
            result = value1 - value2
        case "/":
            result = value1 / value2
        case "*":
            result = value1 * value2
            
    return result
    